packages/expo-modules-core/android/build.gradle

```diff
@@ -422,18 +422,16 @@ task prepareFolly(dependsOn: [downloadFolly], type: Copy) {
 }
 // END FOLLy
 
-task prepareHermes() {
+task prepareHermes(dependsOn: createNativeDepsDirectories, type: Copy) {
   if (!FOR_HERMES) {
     return
   }
 
   def soFiles = zipTree(HERMES_AAR).matching({ it.include "**/*.so" })
 
-  copy {
-    from soFiles
-    from "$REACT_NATIVE_DIR/ReactAndroid/src/main/jni/first-party/hermes/Android.mk"
-    into "$thirdPartyNdkDir/hermes"
-  }
+  from soFiles
```

Comment:
i would like to keep the copy block and remove the `type: Copy`. because i will have a pr to add `doLast`: https://github.com/expo/expo/pull/18472/files#diff-dcab4f253e7efe2ec641357ce996a56297c9b64ed9ef5ce4b2e7ae571263d8afR445